core/src/main/scala/chisel3/internal/firrtl/IR.scala

```diff
@@ -84,6 +85,10 @@ case class Node(id: HasId) extends Arg {
     case Some(arg) => arg.name
     case None => id.instanceName
   }
+  override def simpleName: String = id.getOptionRef match {
```

Comment:
```suggestion
  override private[chisel3] def simpleName: String = id.getOptionRef match {
```
Even though everything else is public (it shouldn't have been), we should keep this package private